railgun/internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -295,13 +300,23 @@ func (p *clientgoCachedProxyResolver) updateKongAdmin() error {
 	// generate the deck configuration to be applied to the admin API
 	targetConfig := deckgen.ToDeckContent(p.ctx, p.deprecatedLogger, kongstate, nil, nil)
 
+	// retrieve custom entities
+	var customEntities []byte
+	if p.kongConfig.InMemory && p.KongCustomEntitiesSecret != "" {
+		customEntities, err = fetchCustomEntities(p.KongCustomEntitiesSecret, storer)
```

Comment:
The language "fetch" may suggest that request external to the Linux process will be performed, but in this implementation it's retrieving from memory right? So might a different name like "getCustomEntitiesFromClientGoCache" (or maybe something similar but shorter) provide helpful insight to the code reader?

Reply:
Added comment instead. It's close by (within the same file) anyway, so I'd prefer that over the long name.